buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -232,6 +233,7 @@ SHORT_NAMES = ['all' : ['ppc64le_linux','s390x_linux','x86-64_linux','ppc64_aix'
             'zoslargeheap' : ['s390x_zos_xl'],
             'zosxlcm' : ['s390x_zos_xl_cm'],
             'zosmxd' : ['s390x_zos_mixed'],
+            'riscv' : ['riscv-64_linux'],
```

Comment:
Will this value `riscv-64_linux` get passed into the test pipeline as `PLATFORM`? If so, it does not match with test pipeline expected value https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/openjdk_tests#L64

Reply:
Updated in the next push